packages/charts/react-charting/src/components/DeclarativeChart/DeclarativeChart.tsx

```diff
@@ -247,7 +247,7 @@ export const DeclarativeChart: React.FunctionComponent<DeclarativeChartProps> =
       return (
         <HeatMapChart
           {...transformPlotlyJsonToHeatmapProps(plotlySchema)}
-          legendProps={legendProps}
+          legendProps={{ onChange: onActiveLegendsChange, canSelectMultipleLegends: false, selectedLegends: [] }}
```

Comment:
heatmapchart doesn't need legendProps. It has hide legends = true

Reply:
Previously, also we were passing legendProps. And now we have to pass empty array to selectedLegends prop.

Reply:
there are no legends in heatmap in declarative chart. Why do you need legend props then?